### PR TITLE
Maintenance update

### PR DESCRIPTION
**Pull request by amycheng**
- updates webpack
- removes mention of using `yarn` in the readme
- first pass at testing

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -51,8 +51,6 @@
     "wrap-iife": 2,
     // strict mode
     "strict": [2, "never"],
-    // variables
-    "no-unused-vars": 2,
```
Comment (fraddski) [inline .eslintrc]: Is there a reason for removing this rule?

File: webpack.config.js [hunk 2]
```diff
@@ -1,34 +1,81 @@
...
     alias: {
       references: path.resolve('./src/services/references.js')
     }
   },
   plugins: [
-    new ExtractTextPlugin('clay-space-edit.css')
-  ]
+    new webpack.EnvironmentPlugin(['NODE_ENV'])
+  ].concat(
+    nodeEnv === 'testing'
```
Comment (fraddski) [inline webpack.config.js]: This check is done a few times, might be worth pulling to the top as an `isTestEnv` flag or similar.

Comment (amycheng) [reply webpack.config.js]: fixed!

File: .eslintrc (hunk 1 above)
Comment (amycheng) [reply .eslintrc]: don't know where this came from, but it's gone from the new eslint config

**Pull request merged**